src/main/resources/db/migration/V202409081445__create_owner_table.sql

```diff
@@ -0,0 +1,21 @@
+create table owner
+(
+    owner_id serial primary key,
```

Comment:
rename to id, you are already in owner table context, so the owner prefix is redundant